### PR TITLE
support.sh: also inspect stopped containers

### PR DESCRIPTION
**Pull request by trapier**
The fate (exit code and timing) of exited containers is often useful in back-tracking swarmkit IPAM issues that manifest in libnetwork gossip control plane.

Collect information on `container ls -a` instead of just `container ls`.

Used a bashim to keep the number of docker API calls the same.

### CODE REVIEW

**Comment by trapier** [comment]
cc @adamancini.

File: support.sh [hunk 1]
```diff
@@ -65,15 +65,17 @@ for networkID in $(${DOCKER} network ls --no-trunc --filter driver=overlay -q) "
 done
 
 echo "Container network configuration"
-for containerID in $(${DOCKER} container ls -q); do
+while read containerID status; do
```
[the remark this answers is not in the document]
Comment (adamancini) [reply support.sh]: Otherwise LGTM

[the remark this answers is not in the document]
Comment (fcrisciani) [reply support.sh]: it's at the bottom: 
```
<(${DOCKER} container ls -a --format '{{.ID}} {{printf "%.2s" .Status}}')
```

[the remark this answers is not in the document]
Comment (adamancini) [reply support.sh]: Shell construct I'm not familiar with.

File: support.sh [hunk 2]
```diff
@@ -65,15 +65,17 @@ for networkID in $(${DOCKER} network ls --no-trunc --filter driver=overlay -q) "
...
     echo "ccc Container ${containerID}"
```
Comment (fcrisciani) [inline support.sh]: would it be useful to put here what is the container state??
`ccc Container xxxxxxxx state: Up/Down/Failed etc`

**Comment by fcrisciani** [comment]
ping @trapier

File: support.sh (hunk 2 above)
Comment (trapier) [reply support.sh]: not to me, since that information is present in the `State` line, four lines later.

Comment (trapier) [reply support.sh]: is there a concern that suddenly logging information on stopped containers will trip up current post-processing?

Comment (fcrisciani) [reply support.sh]: was thinking was just useful to have it at the top

Comment (fcrisciani) [reply support.sh]: it's already there :D, I'm suggesting to just add the string in the header of the list

Comment (trapier) [reply support.sh]: good point. i'll add it.

Comment (trapier) [reply support.sh]: `state: Up/Down/Failed`:  with or without the whitespace after the colon?

Comment (fcrisciani) [reply support.sh]: ccc Container xxxxxxxx state: Up
vs
ccc Container xxxxxxxx state:Up
vs
ccc Container xxxxxxxx state Up

I would vote for the first option, but 3rd option is in line with the fact that we don't have `:` for the container id

Comment (trapier) [reply support.sh]: Going with first option.  Rebased onto master.  Testing in support dump on orca branch `release/v3.0.x` before updating here.  Should be done with that shortly.

**Pull request merged**